apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/EditLimitPrice.tsx

```diff
@@ -70,21 +72,33 @@ type AmountInputProps = {
   value?: string
   onChange(value: string): void
   editable: boolean
+  inputRef?: React.RefObject<TextInput>
 }
-const AmountInput = ({onChange, value, editable}: AmountInputProps) => {
+const AmountInput = ({onChange, value, editable, inputRef}: AmountInputProps) => {
+  const amountInputRef = React.useRef<TextInput>(inputRef?.current ?? null)
+
+  const focusInput = () => {
+    if (amountInputRef?.current) {
+      amountInputRef.current.focus()
+    }
+  }
+
   return (
-    <TextInput
-      keyboardType="numeric"
-      autoComplete="off"
-      value={value}
-      placeholder="0"
-      onChangeText={onChange}
-      allowFontScaling
-      selectionColor={COLORS.TRANSPARENT_BLACK}
-      style={styles.amountInput}
-      underlineColorAndroid="transparent"
-      editable={editable}
-    />
+    <Pressable onPress={focusInput}>
```

Comment:
I know we have this in AmountCard but just noticed it. Is this Pressable thing neccesary, don't TextInputs focus by themselves?